### ISSUE TITLE
Examples not correct for 3.3.2: Labels or Instructions

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1794]
https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html

The following **Examples** are not for this SC as they are about clear and descriptive, not that there is a label present weather or not it makes sense:

> - A field for entering a given name is clearly labeled with "Given Name" and the field for family name is labeled "Family Name" to avoid confusion over which name is requested.
> - A U.S. phone number separates the area code, exchange, and number into three fields. Parentheses surround the area code field, and a dash separates the exchange and number fields. While the punctuation provides visual clues to those familiar with the U.S. telephone number format, the punctuation is not sufficient to label the fields. The single "Phone number" label also cannot label all three fields. To address this, the three fields are grouped in a fieldset with the legend "Phone number". Visual labels for the fields (beyond the punctuation) cannot be provided in the design, so invisible labels are provided with the "title" attribute to each of the three fields. The value of this attribute for the three fields are, respectively, "Area Code", "Exchange", and "Number".

The problem with the examples is of course that they do not have to be "sufficiently clear or descriptive" but that they might also serve as a instruction, thus pushing them in a truly grey area weather it is 3.3.2 OR 2.4.6...

The blur for clear / descriptive and just being present needs to be shown as it will otherwise confuse what is needed exactly.

Based on:

> While this Success Criterion requires that controls and inputs have labels, whether or not these labels are sufficiently clear or descriptive is covered separately by 2.4.6: Headings and Labels.
